ipysurfer/viz/brain.py

```diff
@@ -0,0 +1,541 @@
+import ipyvolume as ipv
+import ipywidgets as widgets
+import numpy as np
+from pythreejs import (BlendFactors, BlendingMode, Equations, ShaderMaterial,
+                       Side)
+
+from .._utils import _calculate_lut, _mesh_edges, _smoothing_matrix
+from .view import views_dict, ColorBar
+from .surface import Surface
+
+
+class Brain:
```

Comment:
classes generally inherit from "object" yeah? e.g. https://github.com/mne-tools/mne-python/blob/601a98f580aafbee3ea7687d91ffda7e83b57af5/mne/decoding/mixin.py#L4

Reply:
As far as I see, that depends on `Python` version. In `Python 3` I can write `class Brain` and it implicitly inherit from `object`. On the other hand, I will need to write `class Brain(object)` for `Python 2`. I was using `Python 3` for this repo, maybe it will be better to make explicit inheritance from `object`, in case we would want this module to work with `Python 2`. What do you think?

Reply:
ah I see what you mean - I think we should try and keep things relatively 2.X compatible for now (though it seems like packages are starting to drop 2.X compatibility, I think MNE will lag behind this by a little bit...besides, I'm pretty sure "explicit is better than implicit" is the second line of [the Zen of Python](https://www.python.org/dev/peps/pep-0020/) :-)  )

Reply:
ok